fastlane/helpers/github.rb

```diff
@@ -0,0 +1,10 @@
+def create_release_management_pull_request(base_branch, title)
+  create_pull_request(
+      api_token: ENV['GITHUB_TOKEN'],
```

Comment:
[Fastlane uses `ENV["GITHUB_API_TOKEN"]` as the default ](https://docs.fastlane.tools/actions/create_pull_request/)whereas we are using `ENV["GITHUB_TOKEN"]`. Instead of adding another environment variable, I thought it'd be better to default to the existing one.

Reply:
Sound 👍

Reply:
This is a good way to go. The reason we use that environment variable is that it's what OctoKit expects FWIW.